### PR TITLE
docs(secret-manager): rework secret manager documentation 

### PR DESCRIPTION
**Pull request by nerda-codes**
### Description

- Reworked the whole documentation to:
** remove outdated screenshots
** update all pages to guarantee consistency with changes made in the console
** add "Region" concept 
- Moved most-viewed tutorial to the API/CLI section

### CODE REVIEW

File: identity-and-access-management/secret-manager/api-cli/external-secrets.mdx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+1. Make sure you are connected to your cluster and that `kubectl` and `helm` are installed on your local machine.
+2. Add the External Secrets helm repo and update it using the following command:
+    ```
+    helm repo add external-secrets https://charts.external-secrets.io
+    helm repo update
+    ```
+
+## Deploying External Secrets
+
+To automatically install and manage the CRDs as part of your Helm release, you must add the `--set installCRDs=true` flag to your Helm installation command.
```
Comment (bene2k1) [inline identity-and-access-management/secret-manager/api-cli/external-secrets.mdx:43]: Maybe write out CRD one time here

File: identity-and-access-management/secret-manager/api-cli/external-secrets.mdx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+Uncomment the relevant line in the next steps to enable this.
+```
+helm upgrade --install external-secrets  external-secrets/external-secrets \
+    -n external-secrets \
+    --create-namespace \
+    # --set installCRDs=true
+```
+
+## Create a secret containing your Scaleway API key information
+
```
Comment (bene2k1) [inline identity-and-access-management/secret-manager/api-cli/external-secrets.mdx:53]: Maybe add here to replace `KEYID` and `SECRETKEY` with personal values

File: identity-and-access-management/secret-manager/how-to/create-folder.mdx [hunk 3]
```diff
@@ -22,20 +22,45 @@ This page explains how to create [folders](/identity-and-access-management/secre
...
 
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
-2. Click **Create folder**.
-3. Enter a name for your folder and click **Create folder**. Your newly created folder displays.
+<Tabs id="install">
+  <TabsTab label="Create your first folder">
 
-## How to create a subfolder
+   <Message type="note">
+     If you have no resources yet in Secret Manager, you can create a folder when creating your secret.
```
Comment (bene2k1) [inline identity-and-access-management/secret-manager/how-to/create-folder.mdx:30]:
```suggestion
     If you have no resources in Secret Manager yet, you can create a folder when creating your secret.
```

File: identity-and-access-management/secret-manager/how-to/create-folder.mdx [hunk 4]
```diff
@@ -22,20 +22,45 @@ This page explains how to create [folders](/identity-and-access-management/secre
...
+    7. Click **Create secret**. The **Overview** tab of your secret displays.
+    8. Click **Back to Secrets** to display your folder and its contents.
+  </TabsTab>
+  <TabsTab label="Create more folders">
+    1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+    2. Select your desired [region](/identity-and-access-management/secret-manager/concepts/#region) in the **Region** drop-down.
+         <Message type="important">
+         Folders cannot be moved from one region to another after creation.
+        </Message>
+    3. Navigate to the [path](/identity-and-access-management/secret-manager/concepts/#path) you want to create your folder in (either `/(root)` or an existing folder).
```
Comment (bene2k1) [inline identity-and-access-management/secret-manager/how-to/create-folder.mdx:57]: the brackets around `/(root)` are correct?

**Review by bene2k1 — APPROVED** [review]

File: identity-and-access-management/secret-manager/api-cli/external-secrets.mdx [hunk 5]
```diff
@@ -7,11 +7,9 @@ content:
   paragraph: Learn how to deploy External Secrets on Kubernetes Kapsule integrated with Scaleway Secret Manager
 tags: kapsule-cluster kubernetes external-secrets secret-management
 categories:
-  - kubernetes
-  - secret-manager
-  - security-identity
+  - identity-and-access-management
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/api-cli/external-secrets.mdx:10]: Why these changes in categories?

File: identity-and-access-management/secret-manager/concepts.mdx [hunk 6]
```diff
@@ -40,6 +40,16 @@ A PEM or **Privacy Enhanced Mail** file is a security certificate file used to e
 
 </Concept>
 
+<Concept>
+
+## Region
+
+A region is the geographical area in which your folders, secrets, and versions are stored and run. They are duplicated on all [Availability Zones](/compute/instances/concepts/#availability-zone) of the selected region (Paris, Amsterdam or Warsaw).
+
+Secret Manager allows you to filter your resources by region. This allows you to better sort through your resources, if you have created them in different regions but with the same name, for example.
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/concepts.mdx:49]: I think we should say select more than filter, because filter implies there is a global view, that does not exist.

File: identity-and-access-management/secret-manager/how-to/create-folder.mdx [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
 dates:
-  validation: 2023-09-07
+  validation: 2023-11-21
   posted: 2023-09-07
 categories:
   - identity-and-access-management
 ---
 
-This page explains how to create [folders](/identity-and-access-management/secret-manager/concepts/#folder) and subfolders using the [Scaleway console](https://console.scaleway.com).
+The [folder](/identity-and-access-management/secret-manager/concepts/#folder) creation process slightly differs depending on wether you are using Secret Manager for the first time, or not. This page explains how to create a [folder](/identity-and-access-management/secret-manager/concepts/#folder) for the first time using the [Scaleway console](https://console.scaleway.com) and how to create a folder if you have already created resources in Secret Manager.
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/how-to/create-folder.mdx:16]: Whether (a h is missing)

File: identity-and-access-management/secret-manager/how-to/create-secret.mdx [hunk 8]
```diff
@@ -1,19 +1,19 @@
...
 dates:
-  validation: 2023-06-16
+  validation: 2023-11-22
   posted: 2023-02-21
 categories:
   - identity-and-access-management
 ---
 
-This page explains how to create a [secret](/identity-and-access-management/secret-manager/concepts/#secret) using the [Scaleway console](https://console.scaleway.com).
+The [secret](/identity-and-access-management/secret-manager/concepts/#secret) creation process slightly differs depending on wether you are using Secret Manager for the first time, or not. This page explains how to create a [secret](/identity-and-access-management/secret-manager/concepts/#secret) for the first time using the [Scaleway console](https://console.scaleway.com) and how to create a secret if you have already created resources in Secret Manager.
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/how-to/create-secret.mdx:16]: Whether

File: identity-and-access-management/secret-manager/how-to/create-secret.mdx [hunk 9]
```diff
@@ -22,22 +22,47 @@ This page explains how to create a [secret](/identity-and-access-management/secr
   - You have [generated an API key](/identity-and-access-management/iam/how-to/create-api-keys/) and enabled the `SecretManagerFullAccess` [permission set](/identity-and-access-management/iam/reference-content/permission-sets/)
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/how-to/create-secret.mdx:22]: Why do I need to have generated an API key? This is just a console use case

Comment (crlptl) [reply identity-and-access-management/secret-manager/how-to/create-secret.mdx:22]: Maybe need to change the requirements?

File: identity-and-access-management/secret-manager/how-to/delete-folder.mdx [hunk 10]
```diff
@@ -23,13 +23,14 @@ This page shows you how to delete a [folder](/identity-and-access-management/sec
...
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets and folders display.
-2. Click the <Icon name="delete" /> icon next to the folder you want to delete.
-3. Type **DELETE** to confirm and click **Delete folder**.
 
-<Message type="important">
-  This will delete all subfolders and secrets you might have created in the folder.
-</Message>
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+  <Message type="important">
+   Your folder must be empty to be deleted. Folders containing resources cannot be deleted.
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/how-to/delete-folder.mdx:29]: I would say "Folders containing other folders or secrets"

File: identity-and-access-management/secret-manager/how-to/manage-secret.mdx [hunk 11]
```diff
@@ -23,14 +23,14 @@ This page explains how to manage your [secret](/identity-and-access-management/s
   - Your have [created a secret](/identity-and-access-management/secret-manager/how-to/create-secret/)
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/how-to/manage-secret.mdx:23]: You have created (not your have)

File: identity-and-access-management/secret-manager/quickstart.mdx [hunk 12]
```diff
@@ -6,11 +6,11 @@ content:
...
 dates:
-  validation: 2023-06-16
+  validation: 2023-11-21
   posted: 2023-02-21
 ---
 
-Scaleway’s [Secret Manager](/identity-and-access-management/secret-manager/concepts/#secret-manager) allows you to conveniently [store, access and share sensitive data](/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/) such as passwords, API keys and certificates. In this quickstart, we show you how to create and manage a secret, how to create and manage a secret's version and how to delete your resources once you are done using them.
+In this quickstart, we show you how to create a [folder](/identity-and-access-management/secret-manager/concepts/#folder) and a [secret](/identity-and-access-management/secret-manager/concepts/#secret) and how to create a secret's [version](/identity-and-access-management/secret-manager/concepts/#version).
 
 <Macro id="iam-requirements" />
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/quickstart.mdx:16]: Again, the requirement is not clear. Since the use case is based on console, I don't hink we need to mention any API key

File: identity-and-access-management/secret-manager/quickstart.mdx [hunk 13]
```diff
@@ -20,93 +20,42 @@ Scaleway’s [Secret Manager](/identity-and-access-management/secret-manager/con
...
-## How to create a version of a secret
-
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets display.
-2. Click the secret you want to add a version to. Your secret's overview page displays.
-3. Click the **Versions** tab.
-4. Click <Icon name="plus" /> and **Create version**. A pop-up displays.
-5. Add your version manually or click <Icon name="plus" /> to import it from a file.
-6. Click <Icon name="toggle" /> to enable or disable the version.
-7. Click **Add new version**.
+6. Click **Create secret**. The **Overview** tab of your secret displays. You can copy your secret's ID, add a description, tags or delete your secret.
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/quickstart.mdx:44]: I would add a message here saying clearly that:
- A secret has been created
- If the path contained folder that did not exist yet, all of them are created on the go
- the value of the secret is stored in the first version of this secret, which is enabled by default

File: identity-and-access-management/secret-manager/quickstart.mdx [hunk 14]
```diff
@@ -20,93 +20,42 @@ Scaleway’s [Secret Manager](/identity-and-access-management/secret-manager/con
...
-    * Enter a name for your secret.
-    * Optionally, add tags.
-    * Add your secret manually or click <Icon name="plus" /> to import it from a file.
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+2. Click **+ Create secret**.
+3. Choose the [region](/identity-and-access-management/secret-manager/concepts/#region) in which you want to store your folder and your secret.
+    <Message type="important">
+     Folders and secrets cannot be moved from one region to another after creation.
+    </Message>
+4. Select **Create a new path** and enter `/`, followed by a name for your [folder](/identity-and-access-management/secret-manager/concepts/#folder).
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/quickstart.mdx:35]: There is probably no folder yet so "your" folder is a bit confusing to me

File: identity-and-access-management/secret-manager/quickstart.mdx [hunk 15]
```diff
@@ -20,93 +20,42 @@ Scaleway’s [Secret Manager](/identity-and-access-management/secret-manager/con
...
-## How to delete a folder
+    <Message type="tip">
+      - Read our [documentation](/identity-and-access-management/secret-manager/how-to/create-folder/) to learn how to create more folders once you have created resources in Secret Manager.
+      - Read our [documentation](/identity-and-access-management/secret-manager/api-cli/migrate-secrets-in-folders/) to find out how to migrate secrets located in the `/` path, to specific folders.
+    </Message>
 
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets and folders display.
-2. Click the <Icon name="delete" /> icon next to the folder you want to delete.
-3. Type **DELETE** to confirm and click **Delete folder**.
+## How to create a secret version
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/quickstart.mdx:52]: Maybe we can say that at its creation, a secret has one version, and that the user can add MORE

File: identity-and-access-management/secret-manager/quickstart.mdx [hunk 16]
```diff
@@ -20,93 +20,42 @@ Scaleway’s [Secret Manager](/identity-and-access-management/secret-manager/con
...
 
-<Message type="important">
-  This will delete all subfolders and secrets you might have created in the folder.
-</Message>
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your resources located in the `/` path of the Paris region display.
+2. In the **Region** drop-down, select the pertinent [region](/identity-and-access-management/secret-manager/concepts/#region) in which to create your secret version.
```
Comment (crlptl) [inline identity-and-access-management/secret-manager/quickstart.mdx:55]: "in which the secret you want to update is". We do not create secret versions, we add versions to an existing secret (otherwise it's a full secret creation)

**Review by crlptl — COMMENTED** [review]
Good job! I tried to read eveything in detail because I don't know this doc a lot - hence the important number of feedbacks. Hope you will not hate tinky winky after this

File: identity-and-access-management/secret-manager/how-to/create-folder.mdx [hunk 17]
```diff
@@ -22,20 +22,45 @@ This page explains how to create [folders](/identity-and-access-management/secre
...
+         The maximum file size for your secret is 64 KiB.
+        </Message>
+    7. Click **Create secret**. The **Overview** tab of your secret displays.
+    8. Click **Back to Secrets** to display your folder and its contents.
+  </TabsTab>
+  <TabsTab label="Create more folders">
+    1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+    2. Select your desired [region](/identity-and-access-management/secret-manager/concepts/#region) in the **Region** drop-down.
+         <Message type="important">
+         Folders cannot be moved from one region to another after creation.
```
[the remark this answers is not in the document]
Comment (nerda-codes) [reply identity-and-access-management/secret-manager/how-to/create-folder.mdx:55]: Yes, this is because they will only display when you click the relevant tab. So in reality, the message only displays once per tab. 
Capture d’écran 2023-11-23 à 10 55 49
Capture d’écran 2023-11-23 à 10 55 41

[the remark this answers is not in the document]
Comment (crlptl) [reply identity-and-access-management/secret-manager/how-to/create-folder.mdx:55]: Ok perfect then 👍

**Review by ldecarvalho-doc — APPROVED** [review]
